### PR TITLE
deadbeef: Update to 1.8.8

### LINKED ISSUE #8027: DeadBeeF has moved to sourceforge, need to update
I have updated the json locally. It works.
pic

I could make a pr if approved.

### PR DESCRIPTION
**Pull request by mxarcher**
Closes #8027

- [x] I have read the [Contributing Guide](https://github.com/ScoopInstaller/.github/blob/main/.github/CONTRIBUTING.md).

### CODE REVIEW

File: bucket/deadbeef.json [hunk 1]
```diff
@@ -1,33 +1,33 @@
...
-    "version": "2020-07-06",
-    "description": "Unofficial port of DeaDBeeF music player",
-    "homepage": "https://deadbeef-for-windows.github.io/",
-    "license": {
-        "identifier": "BSD-3-Clause,...",
-        "url": "https://github.com/DeaDBeeF-for-Windows/deadbeef/blob/master/COPYING"
-    },
+    "version": "1.8.8",
+    "homepage": "https://deadbeef.sourceforge.io",
+    "description": "DeaDBeeF (as in 0xDEADBEEF) is a modular cross-platform audio player which runs on GNU/Linux distributions, macOS, Windows, *BSD, OpenSolaris, and other UNIX-like systems.",
```
Comment (rashil2000) [inline bucket/deadbeef.json:4]:
```suggestion
    "description": "DeaDBeeF (as in 0xDEADBEEF) is a modular cross-platform audio player.",
```

File: bucket/deadbeef.json [hunk 2]
```diff
@@ -1,33 +1,33 @@
...
     "shortcuts": [
         [
             "deadbeef.exe",
             "DeaDBeeF"
         ]
     ],
+    "extract_dir": "deadbeef-x86_64",
+    "persist": [
+        "config"
+    ],
```
Comment (rashil2000) [inline bucket/deadbeef.json:22]:
```suggestion
    "persist": "config",
```

**Review by rashil2000 — CHANGES_REQUESTED** [review]

File: bucket/deadbeef.json [hunk 3]
```diff
@@ -1,33 +1,33 @@
...
     "architecture": {
         "64bit": {
-            "url": "https://github.com/DeaDBeeF-for-Windows/deadbeef/releases/download/2020-07-06/deadbeef-2020-07-06-windows-x86_64.exe",
-            "hash": "481393f55fbc78b2db16c412769159bfe5580245974a42c5543fe95ee737a66e"
+            "url": "https://downloads.sourceforge.net/project/deadbeef/travis/windows/1.8.8/deadbeef-1.8.8-windows-x86_64.zip",
+            "hash": "sha1:2f05578852668599319746b38753668fd69a72b8"
         }
     },
-    "innosetup": true,
     "bin": "deadbeef.exe",
```
[the remark this answers is not in the document]
Comment (mxarcher) [reply bucket/deadbeef.json:12]: Sorry, I forget to check what innosetup mean.

[the remark this answers is not in the document]
Comment (rashil2000) [reply bucket/deadbeef.json:12]: In that case the `bin` entry can be removed

[the remark this answers is not in the document]
Comment (mxarcher) [reply bucket/deadbeef.json:12]: OK, I'll remove it.

**Pull request merged**